openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -32,7 +32,7 @@ <h1>Merge duplicate editions</h1>
 
 {{ pager() }}
 
-{{ total }} items need to be merged ({{ easy }} easy)<p>
+{{ total }} items need to be merged ({{ easy }} easy)</p>
```

Comment:
Where is the opening p tag? 
It looks like the <p> inside the macro pager() is illplaced to me. Should that be on this line as well?

e.g. `<p> {{ total }} items need to be merged ({{ easy }} easy)</p>`

Reply:
```
<p>
{% endmacro %}

{{ pager() }}

{{ total }} items need to be merged ({{ easy }} easy)<p>
```
line 2 in https://github.com/internetarchive/openlibrary/blob/master/openlibrary/catalog/edition_merge/templates/index.html#L28
the p tag is opened at line 30 but not closed anywhere it was reopened 
so I closed it not a scope for this pr but I found it and replaced it :D

Reply:
I guess the p tag at line 30 is illegal and it must be this 
```<p> {{ total }} items need to be merged ({{ easy }} easy)</p>```
as you said 
I am thinking of reverting this back I will create a different pr for this